plugins/MantisCoreFormatting/files/markdown.css

```diff
@@ -0,0 +1,10 @@
+/* quotes */
+.bug-description blockquote,
+.bug-steps-to-reproduce blockquote,
+.bug-additional-information blockquote,
+.bugnote-note blockquote {
+    padding: 0.13em 1em;
+    color: rgb(119,119,119);
+    border-left: 0.25em solid #C0C0C0;
+    font-size: 13px;
```

Comment:
I think it would be better to use `font-size: inherit;` - markdown may be used in other places where we're not at 13px.